File: location_service/LocationService.Infrastructure/Repository/Model/LocationModel.cs
namespace LocationService.Infrastructure.Repository.Model
{
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;

    [Table("locations")]
    public class LocationModel : AuditModel
    {
        [Key]
        [Column("id")]
        public long Id { get; set; }

        [Column("country_id")]
        public long CountryId { get; set; }

        [Column("province_id")]
        public long ProvinceId { get; set; }

        [Column("latitude")]
        public double Latitude { get; set; }

        [Column("longitude")]
        public double Longitude { get; set; }

        [Column("detail")]
        public string Detail { get; set; } = string.Empty;

        [Column("name")]
        public string Name { get; set; } = string.Empty;
    }
}

File: profile_service/src/main/java/huy/project/profile_service/infrastructure/repository/model/LocationModel.java
package huy.project.profile_service.infrastructure.repository.model;

import jakarta.persistence.*;
import lombok.*;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
@Entity
@Table(name = "locations")
public class LocationModel extends AuditTable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    @Column(name = "country_id")
    private Long countryId;
    @Column(name = "province_id")
    private Long provinceId;
    @Column(name = "longitude")
    private Double longitude;
    @Column(name = "latitude")
    private Double latitude;
    @Column(name = "address")
    private String address;
}
